ci_scripts/ci_post_clone.sh

```
@@ -1,4 +1,8 @@
-git clone https://github.com/baekteun/TodayWhat-XCConfig.git
+git clone https://github.com/todaywhat/TodayWhat-Secret.git
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)